src/ansiblelint/config.py

```diff
@@ -192,27 +192,31 @@ def parse_ansible_version(stdout: str) -> tuple[str, str | None]:
 
 def in_venv() -> bool:
     """Determine whether Python is running from a venv."""
-    if hasattr(sys, "real_prefix"):
+    if (
+        hasattr(sys, "real_prefix")
+        or os.environ.get('CONDA_EXE', None) is not None
+    ):
```

Comment:
If the commit is about package name, why is this change here ?

Reply:
Because it looked from the path comparison logic like the code didn't take into consideration use of Conda as a virtual environment (though it is). I wasn't running in VScode, so maybe "real_prefix" is in `sys` with Conda?

Reply:
Oh, I was not commenting the change by itself (since I'm not sure to understand it), just that imho, it was deserving its own commit.